### PR TITLE
Updates for including data object names in error messages and enhance test suite with additional table types

### PR DESCRIPTION
**Pull request by al-niessner**
## 🗒️ Summary
Brief summary of changes if not sufficiently described by commit messages.

## ⚙️ Test Data and/or Report
One of the following should be included here:
* Reference to regression test included in code (preferred wherever reasonable)
* Attach test data here + outputs of tests

## ♻️ Related Issues
#534
#343 
#217 
#190

### CODE REVIEW

File: src/test/resources/features/validate.feature [hunk 1]
```diff
@@ -205,6 +207,8 @@ Scenario Outline: Execute validate command for tests below.
 # https://github.com/NASA-PDS/validate/issues/188 As a user, I want to validate a bundle that uses multiple versions of the Information Model / Discipline LDDs
  |"NASA-PDS/validate#188 VALID" | "github188" | 0 | "0 errors message expected" | "totalErrors" | "src/test/resources" | "target/test" | "--skip-content-validation -r {reportDir}/report_github188_label_valid_both.json  -s json -t {resourceDir}/github188/bundle_cassini-huygens-coradar.xml {resourceDir}/github188/BILQH07S314_D065_T008S02_V02_without_Missing_Area_tag.xml" | "report_github188_label_valid_both.json" |
 
+ |"NASA-PDS/validate#217 Success with Errors: Tables Types" | "github217" | 15 | "15 error message expected" | "totalErrors" | "src/test/resources" | "target/test" | "-r {reportDir}/report_github217.json  -s json -t {resourceDir}/github217" | "report_github217.json" |
```
Comment (jordanpadams) [inline src/test/resources/features/validate.feature:210]: @al-niessner can we break out the success use cases vs. the failures just so we are more explicit?

**Review by jordanpadams — CHANGES_REQUESTED** [review]
See requested change to validate.feature

File: src/test/resources/features/validate.feature (hunk 1 above)
Comment (al-niessner) [reply src/test/resources/features/validate.feature:210]: @jordanpadams 

Can but it is going to make the features file even more ungainly. Cannot wildcard in cucumber so go from one line to 20+ lines. This has been fixed for a while so not sure this is insufficient. They all share the same tables so even breaking up the directory to keep using validates directory processing would be odd at best for the next person who needs to look at it.

Really want it broken apart?

Comment (jordanpadams) [reply src/test/resources/features/validate.feature:210]: @al-niessner copy that. let's keep it as-is for now and we can break it out on a case-by-case basis down the road if specific use cases prove troublesome for whatever reason

**Review by jordanpadams — APPROVED** [review]

**Pull request merged**